<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editticketconfig</class>
 <widget class="QDialog" name="Editticketconfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>501</width>
     <height>371</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="LblMensagem">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>203</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Mensagem</string>
    </property>
   </widget>
   <widget class="QLabel" name="LblDescription">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LnEdtDescription">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>411</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>300</y>
      <width>480</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string> Financeiro R$</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>451</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="LblJuros">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Juros</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxJuros"/>
      </item>
      <item>
       <widget class="QLabel" name="LblJuros_2">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Multa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxMulta"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTextEdit" name="textEditMensagem">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>411</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>480</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>  Configurações </string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>461</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="Lblespecie">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Especie</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SComboBox" name="CmbBxespecie">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>EC</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DM</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LblNossoNumero">
        <property name="text">
         <string>Nosso Número</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="DblSpnBxNossoNumero">
        <property name="maximum">
         <double>999999999999.989990234375000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LblCarteira">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Carteira</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LnEdtCarteira">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>461</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="LblRespEmis">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Emitente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SComboBox" name="CmbBxRespEmis">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>ClienteEmite</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BancoEmite</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BancoReemite</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BancoNãoReemite</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LblTipoPessoa">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Pessoa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SComboBox" name="CmbBxTipoPessoa">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Física</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Juridica</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Outros</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LblCodigoCedente">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Cedente</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LnEdtCodigoCedente">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>461</width>
       <height>22</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="LblLayoutBol">
        <property name="text">
         <string>Layout</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SComboBox" name="CmbBxLayoutBol">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Padrão</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Carnê</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fatura</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PadrãoEntrega</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="LblTipoCobranca">
        <property name="text">
         <string>Tipo Cobrança</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SComboBox" name="CmbBxTipoCobranca">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Simples</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vinculada</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Caucionada</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Descontada</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vendor</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="Lblcnab">
        <property name="text">
         <string>CNAB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="SComboBox" name="CmbBxcnab">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>CNAB240</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>CNAB400</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QFrame" name="topmenu">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>801</width>
     <height>42</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>42</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>42</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 0, 133);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="labelStatus">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>0</y>
      <width>541</width>
      <height>39</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>50</red>
          <green>0</green>
          <blue>133</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Configuração de Boleto</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="PshBtnCancel">
   <property name="geometry">
    <rect>
     <x>336</x>
     <y>430</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PshBtnSave">
   <property name="geometry">
    <rect>
     <x>423</x>
     <y>430</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Salvar</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">scombobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>LnEdtDescription</tabstop>
  <tabstop>CmbBxRespEmis</tabstop>
  <tabstop>CmbBxTipoPessoa</tabstop>
  <tabstop>LnEdtCodigoCedente</tabstop>
  <tabstop>CmbBxLayoutBol</tabstop>
  <tabstop>CmbBxTipoCobranca</tabstop>
  <tabstop>CmbBxcnab</tabstop>
  <tabstop>CmbBxespecie</tabstop>
  <tabstop>DblSpnBxNossoNumero</tabstop>
  <tabstop>LnEdtCarteira</tabstop>
  <tabstop>textEditMensagem</tabstop>
  <tabstop>doubleSpinBoxJuros</tabstop>
  <tabstop>doubleSpinBoxMulta</tabstop>
  <tabstop>PshBtnCancel</tabstop>
  <tabstop>PshBtnSave</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
